internal/manifest/readycheck/statefulset_ready_check.go

```diff
@@ -1,10 +1,13 @@
-package manifest
+package readycheck
```

Comment:
Just to confirm my understanding. We are going to tackle the stateful set part iwth the other ticket you created. Here the changes are necessary because we removed the helper funcs from the `ready_check.go`?

Reply:
Yes I moved the GetPodsState here because right now it is only used for the StatefulSet and so that it would be easier to remove this when we enhance the state determination for StatefulSets.